fixtures/run_as_non_root_psc_true_csc_false_v1.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
This is actually okay. Because CSC wins over PSC.
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#podsecuritycontext-v1-core

> Indicates that the container must run as a non-root user. If true, the Kubelet will validate the image at runtime to ensure that it does not run as UID 0 (root) and fail to start the container if it does. If unset or false, no such validation will be performed. May also be set in SecurityContext. If set in both SecurityContext and PodSecurityContext, the value specified in SecurityContext takes precedence.